src/components/Buy/column.js

```diff
@@ -0,0 +1,47 @@
+// File to specify the columns in the datatable
+
+export const noBuyColumns = [
+  {
+    label: 'Course Number',
+    field: 'courseAndDpmt',
```

Comment:
If the data includes the department and course number, shouldnt the label just be `Course`?

Reply:
the Course just has the numbering of the class. But for clarity we want to show the department and the course number. 

That is we want to show MATH32 instead of 32 and MATH in two seperate columns.